situp-plugins/elasticsearch/README.md

```diff
@@ -12,6 +12,9 @@ pipeline:
   sink:
     elasticsearch:
       hosts: ["http://localhost:9200"]
+      index_type: raw
+      index_alias: "apm-span"
+      template_file: /your/local/template-file.json
```

Comment:
So, the users should not give these for trace-analytics feature. We should support configs like below,

```
trace_analytics_raw: true
or 
trace_analytics_service_map: true
```
Users can set only one of these configs to true. If they do, we should default the configs accordingly.